text/0000-official-utility-buildpacks.md

```diff
@@ -46,6 +46,12 @@ Examples of potential utility buildpacks include:
 
 The utility buildpacks will be owned, supported, and maintained by the [Implementation team](https://github.com/buildpacks/community/blob/main/TEAMS.md#implementation-team).
 
+The criteria for accepting a utility buildpack into the project is as follows:
+- The behavior provided by the buildpack MUST be defined in the spec or an extension spec.
+- The buildpack MUST support arbitrary stacks
+- The buildpack MUST support the latest API version
+- The buildpack MUST include detection logic (i.e. it does not always pass)
```

Comment:
👍 

Question regarding the last item. It seems a little strict. Can you elaborate on the rationale for this? Is this requirement to only allow buildpacks that can be disabled in some way?

I have seen a number of buildpacks that follow the pattern of always pass, but restrict behavior in other ways (based on bindings, env variables, or buildplan entries). 

Would it be sufficient to say `The buildpack MUST include a way to be disabled` instead?

Reply:
yeah, I'm curious about the thought process behind this one.

Reply:
I didn't pull this from a specific example. But I suspect that something that _always_ passes detection is something that we might want to have as a native behavior (not a buildpack). `The buildpack MUST include a way to be disabled` seems like a good alternative though.